.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
```

Comment:
Nit; As I had the doubt and had to double-check the docs, do you think it worths adding a comment here stating that `gomod` manages vendor by default, and thus `vendor: true` is not needed?

Reply:
I did check this indeed :)

> Don't use this option if you're using `gomod` as Dependabot automatically detects vendoring for this tool.

Reply:
Exactly, that's why I was wondering whether or not would make sense to leave a comment, for future readers.